programs/bpf_loader/src/lib.rs

```diff
@@ -180,9 +180,6 @@ pub fn load_program_from_account(
         return Err(InstructionError::IncorrectProgramId);
     }
 
-    let (programdata_offset, deployment_slot) =
-        get_programdata_offset_and_depoyment_offset(&log_collector, program, programdata)?;
-
     if let Some(ref tx_executor_cache) = tx_executor_cache {
         if let Some(loaded_program) = tx_executor_cache.get(program.get_key()) {
             if loaded_program.is_tombstone() {
```

Comment:
We can have tombstone in the TX batch cache now, even if `delay_visibility_of_program_deployment` is not enabled, right?

Reply:
Yes, in case the program fails to compile/verify.

Reply:
I was referring to the comment below which is incorrect now.